modules/javafx.graphics/src/main/java/com/sun/prism/mtl/MTLPhongMaterial.java

```diff
@@ -82,7 +82,7 @@ private Texture setupTexture(TextureMap map, boolean useMipmap) {
         if (texture instanceof MTLTexture mtlTex) {
             hTexture = mtlTex.getNativeHandle();
         }
-        context.setMap(nativeHandle, map.getType().ordinal(), hTexture);
+        context.setMap(nativeHandle, map.getType().ordinal(), hTexture, useMipmap);
```

Comment:
We use mipmapping only when `(!PlatformUtil.isEmbedded()) && (i == PhongMaterial.DIFFUSE || i == PhongMaterial.SELF_ILLUM);`. We need to pass mipmap information, so that we can pick appropriate sampler state in native code.